### PR TITLE
VB-57: Feature/create header

### PR DESCRIPTION
**Pull request by mishabenq**
Just for review, what else needs to be done

### CODE REVIEW

File: packages/frontend/components/header/component.tsx [hunk 1]
```diff
@@ -9,20 +10,60 @@ export const Header = () => (
...
+            justifyContent: 'space-between',
+            width: '70%',
+            gap: '1em',
+          }}
+        >
+          <InternalLink href={Routes.DEFAULT} label="Home" />
+          <InternalLink href={Routes.DEFAULT} label="Category" />
+          <InternalLink href={Routes.DEFAULT} label="Search" />
+          <InternalLink href={Routes.DEFAULT} label="News" />
+          <InternalLink href={Routes.DEFAULT} label="About us" />
```
Comment (Nightlngale) [inline packages/frontend/components/header/component.tsx:45]: Please use localised values. Update namespace common.

header: {
  nav: {
    ...
}
}

File: packages/frontend/components/header/component.tsx [hunk 2]
```diff
@@ -9,20 +10,60 @@ export const Header = () => (
...
+          marginLeft: '4em',
+        }}
+      >
+        <nav
+          style={{
+            display: 'flex',
+            alignItems: 'center',
+            justifyContent: 'space-between',
+            width: '70%',
+            gap: '1em',
```
Comment (Nightlngale) [inline packages/frontend/components/header/component.tsx:38]: lets use css

File: packages/frontend/components/header/component.tsx [hunk 3]
```diff
@@ -9,20 +10,60 @@ export const Header = () => (
...
-        <Link href={Routes.USERS}>
-          <a>users</a>
-        </Link>
-      </nav>
+      <Container
+        style={{
+          display: 'flex',
+          alignItems: 'center',
+          flexGrow: 2,
+          maxWidth: '60%',
```
Comment (Nightlngale) [inline packages/frontend/components/header/component.tsx:28]: not sure about flexGrow: 2 and maxWidth 60%

File: packages/frontend/components/primitives/logo/component.tsx [hunk 4]
```diff
@@ -0,0 +1,3 @@
+import * as styles from './styles';
+
+export const Logo = () => <span css={styles.logo}></span>;
```
Comment (Nightlngale) [inline packages/frontend/components/primitives/logo/component.tsx:3]: Let's use next/image

**Review by Nightlngale — CHANGES_REQUESTED** [review]

File: packages/frontend/components/header/component.tsx [hunk 5]
```diff
@@ -1,82 +1,195 @@
...
+        <Avatar firstName="John" lastName="Doe" />
+      </Container>
+      <Link href={Routes.USERS}>
+        <a>users</a>
+      </Link>
+    </header>
+  );
+};
+
+export const HeaderLoggedOut = () => {
```
Comment (Nightlngale) [inline packages/frontend/components/header/component.tsx:121]: Use 1 header

File: packages/frontend/components/header/styles.ts [hunk 6]
```diff
@@ -1,7 +1,113 @@
...
+
+export const buttons_wrapper = css`
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  gap: 2em;
+  flex-shrink: 0;
+`;
+
+export const button_create_account_text = (theme: Theme) => css`
```
Comment (Nightlngale) [inline packages/frontend/components/header/styles.ts:85]: use camelCase

**Review by Nightlngale — CHANGES_REQUESTED** [review]

**Comment by mishabenq** [comment]
image
image
image

**Review by Nightlngale — APPROVED** [review]

**Pull request merged**